src/pages/MetricDetail.svelte

```diff
@@ -206,22 +206,24 @@
     <table>
       <col />
       <col />
-      {#if selectedAppVariant.etl.glam_url}
-        <tr>
-          <td>
-            GLAM
-            <HelpHoverable
-              content={"View this metric in the Glean Aggregated Metrics (GLAM) dashboard"}
-              link={"https://docs.telemetry.mozilla.org/cookbooks/glam.html"}
-            />
-          </td>
-          <td>
+      <tr>
+        <td>
+          GLAM
+          <HelpHoverable
+            content={"View this metric in the Glean Aggregated Metrics (GLAM) dashboard"}
+            link={"https://docs.telemetry.mozilla.org/cookbooks/glam.html"}
+          />
+        </td>
+        <td>
+          {#if selectedAppVariant.etl.glam_url}
             <AuthenticatedLink href={selectedAppVariant.etl.glam_url}>
               {params.metric}
             </AuthenticatedLink>
-          </td>
-        </tr>
-      {/if}
+          {:else}
+            GLAM doesn't support <code>{metric.type}</code> metrics yet.
```

Comment:
"yet" implies that GLAM will support events soon. But as far as I know that's not true since events can hold many things and we don't know how to aggregate events in a general case. Do we have a plan for events?

Reply:
you're right, I don't think supporting events is on the roadmap. I'll remove it